### PR TITLE
Homepage dupes.fix

### PR DESCRIPTION
**Pull request by markcyen**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [ ] CircleCI build is passing
- [x] All tests passing locally

## Pull request type
Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
Shows duplicate cocktails in user homepage

Issue Number: 
closes #

## What is the new behavior?
Restricts showing duplicate cocktails on user homepage.

- 
- 
- 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by zachjamesgreen — APPROVED** [review]

File: app/models/cocktail.rb [hunk 1]
```diff
@@ -14,9 +14,10 @@ def self.dashboard_five(_user_id)
       .limit(5)
   end
 
-  def self.sample_rated(sample_size)
+  def self.sample_rated(user_id, sample_size)
     joins(:ratings)
       .select('cocktails.*, ratings.stars as rating')
+      .where('user_id = ?', user_id)
       .sample(sample_size)
```
Comment (mkrumholz) [inline app/models/cocktail.rb:21]: @markcyen  Quick thought here: If we want only top-rated cocktails we could just take out the sample line and add something like this instead: 
```rb
.order('rating desc')
.limit(5)
```

**Review by mkrumholz — APPROVED** [review]
@markcyen I don't know what's breaking about this from your local still -- guessing it's something with DSL for the application.yml or something? Definitely something yelp related. Suggested a change to get only top-rated cocktails while we're at it, but beyond that, this seems to work fine.

File: app/models/cocktail.rb (hunk 1 above)
Comment (markcyen) [reply app/models/cocktail.rb:21]: This could work. Makes it easier rather than passing in a second argument. Do you want me to make this change?

Comment (markcyen) [reply app/models/cocktail.rb]: Updated!

**Pull request merged**